app/script/backup/BackupRepository.js

```diff
@@ -48,15 +49,13 @@ z.backup.BackupRepository = class BackupRepository {
     };
   }
 
-  exportBackup() {
-    return this.backupService.getHistoryCount().then(numberOfRecords => {
-      const userName = this.userRepository.self().username();
+  cancelBackup() {
+    amplify.publish(z.event.WebApp.BACKUP.EXPORT.CANCEL);
```

Comment:
[EDIT] this doesn't seem to be called. If so, it can be removed :)

----
Is it possible to trigger this event from the `viewModel` side?

I feel this kind of stuff doesn't realy belong in a repo/service.